code/datums/martial.dm

```diff
@@ -21,6 +21,9 @@
 /datum/martial_art/proc/grab_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
 	return 0
 
+/datum/martial_art/proc/throw_impact_act(atom/hit_atom, datum/thrownthing/throwingdatum)
+	return//if you do not want this to continue on to throw_impact, then return anything not null (see dropkick for an example)
```

Comment:
Don't if conditions fail on any falsey value, like `null`, `""`, `0`, and `FALSE` (which is 0)?

Reply:
better way to word that thanks

Reply:
```suggestion
	return//if you want this to continue on to throw_impact, then return TRUE
```